mirage/factories/customer-resource.js

```diff
@@ -7,7 +7,14 @@ export default Factory.extend({
   withTitle: trait({
     afterCreate(customerResource, server) {
       let title = server.create('title');
-      customerResource.update('title', title);
+      customerResource.title = title;
```

Comment:
why'd we ditch `update` here?  we're using it further down in the file.  any reason not to be consistent?